lib/facade_converter.ts

```diff
@@ -93,21 +118,31 @@ export class FacadeConverter extends base.TranspilerBase {
   }
 
   private getFileAndName(symbol: ts.Symbol): {fileName: string, qname: string} {
-    while (symbol.flags & ts.SymbolFlags.Alias) symbol = this.tc.getAliasedSymbol(symbol);
-    if (!symbol.valueDeclaration) return null;
+    while (symbol.flags & ts.SymbolFlags.Alias) {
```

Comment:
oh, leftover from a debugging state where I added a statement in there, will revert